src/main/php/peer/CryptoSocket.class.php

```diff
@@ -8,6 +8,40 @@
  */
 class CryptoSocket extends Socket {
   const CTX_WRP = 'ssl';      // stream context option key
+  protected $crpytoImpl= null;
+
+  /**
+   * Connect, then enable crypto
+   * 
+   * @param   float  $timeout
+   * @return  bool
+   * @throws  peer.SocketException
```

Comment:
Maybe we can add `@throws peer.SSLUnverifiedPeerException` and `@throws peer.SSLHandshakeException` here explicitely.